### ISSUE TITLE
the key to show/hide the winrate suggestion stones

### ISSUE THREAD

**Issue by PARKTAIJE** [featurecat/lizzie#443]
Hi!
Thank you for the Lizzie project.

And I think it's better if there is the key to show/hide the winrate suggestion stones.
I think It could help more powerful thinking of playing games.
Now, pondering off is not hiding the winrate suggestions

Thanks.

**Comment by kaorahi** [comment]
z or Z ? (cf. #317, #223, #157)

**Comment by PARKTAIJE** [comment]
Yes! ,it is.
The key is what I wanted.
I didn't know the key z, Z exist..
I couldn't find it when holding x.
thank you, kaorash.

**Comment by PARKTAIJE** [comment]
> z or Z ? (cf. #317, #223, #157)

**Comment by featurecat** [comment]
good idea - we could add that inverted option to the Config. I think a lot of people will like it. also we must add z to the x menu (which is getting so crowded)

**Comment by PARKTAIJE** [comment]
Thanks, kaorahi!
I have applied your commits about key Z,z.
It worked greatly.
Thank you very much, kaorahi!
